src/config.inc

```diff
@@ -32,10 +32,9 @@ endif
 #MINISAT2 = ../../minisat-2.2.1
 #IPASIR = ../../ipasir
 #GLUCOSE = ../../glucose-syrup
-#CADICAL = ../../cadical
 
 # select default solver to be minisat2 if no other is specified
-ifeq ($(BOOLEFORCE)$(CHAFF)$(GLUCOSE)$(IPASIR)$(LINGELING)$(MINISAT)$(MINISAT2)$(PICOSAT)$(CADICAL),)
```

Comment:
Why did this need to be deleted?

It seems to also be containing `minisat2` which is the default.

Reply:
Since cadical is now always included, it was necessary to remove it from the list so that we don't prevent minisat from getting built by default as well. That way, if the build is done with `-Dsat_impl=glucose` for example, then glucose will be built in place of minisat (but cadical is always built regardless of the options).